src/views/consent/ConsentBeacon.js

```diff
@@ -31,15 +31,20 @@ define(['okta'], function (Okta) {
       </div>\
       <div class="logo-wrapper consent-beacon-client">\
         <div class="consent-beacon-border"/>\
-          {{#if clientLogo}}\
-            <img class="client-logo" src="{{clientLogo}}" />\
+          {{#if customLogo}}\
+            <img class="client-logo custom-logo" src="{{customLogo}}" />\
+          {{else}}\
+            <img class="client-logo default-logo" src="{{defaultLogo}}" />\
           {{/if}}\
         </div>\
       </div>\
     ',
 
     getTemplateData: function () {
-      return { clientLogo: this.options.appState.get('targetLogo') && this.options.appState.get('targetLogo').href };
+      return {
+        customLogo: this.options.appState.get('targetLogo') && this.options.appState.get('targetLogo').href,
+        defaultLogo: this.options.appState.get('baseUrl') + '/img/logos/default.png'
```

Comment:
looking at this again... sorry I didn't see this earlier, but can we move this to the appState model as a derived property? so here we can do
```
 defaultLogo: this.options.appState.get('defaultAppLogo');
```
and we can have '/img/logos/default.png' as a constant there in AppState, similar to other image files we have there.

Reply:
Sure thank you for pointing this out, I wasn't actually sure if this was clean